.github/workflows/test-connect.yml

```
@@ -83,6 +83,10 @@ jobs:
         id: set-matrix-legacy-firmware
         run: echo "legacyFirmwareMatrix=$(node ./scripts/ci/connect-test-matrix-generator.js --model=T2T1 --firmware=2.3.0 --env=all --groups=all --cache_tx=false --transport=Bridge)" >> $GITHUB_OUTPUT
 
+      - name: Set latest devices matrix
+        id: set-matrix-latest-firmware
+        run: echo "latestFirmware=$(node ./scripts/ci/connect-test-matrix-generator.js --model=T2T1 --firmware=2-latest --env=all --groups=all --cache_tx=false --transport=Bridge)" >> $GITHUB_OUTPUT
```

```suggestion
        run: echo "latestFirmwareMatrix=$(node ./scripts/ci/connect-test-matrix-generator.js --model=T2T1 --firmware=2-latest --env=all --groups=all --cache_tx=false --transport=Bridge)" >> $GITHUB_OUTPUT
```
Variable name doesn't match